core/src/main/java/jenkins/util/xstream/XStreamDOM.java

```diff
@@ -308,8 +308,10 @@ public String xpath() {
                         count++;
                 boolean more = false;
                 for (int i=pos; !more && i<node.children.size(); i++)
-                    if (node.children.get(i).tagName.equals(child.tagName))
+                    if (node.children.get(i).tagName.equals(child.tagName)) {
                         more = true;
+                        break;
```

Comment:
The `more` is used in the `for` loop condition, so no need to scare the static code analysis :wink: